code/modules/client/preference_setup/loadout/items/augments.dm

```diff
@@ -371,3 +371,19 @@
 	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION
 	origin_restriction = list(/singleton/origin_item/origin/galatea)
 	whitelisted = list(SPECIES_HUMAN, SPECIES_HUMAN_OFFWORLD)
+
+/datum/gear/augment/geneboosted_heart
+	display_name = "Geneboosted Heart"
+	description = "A heart that has been enhanced by Imperial geneboosting to function at the level of a peak human athlete."
+	path = /obj/item/organ/internal/heart/geneboosted_heart
+	origin_restriction = list(/singleton/origin_item/origin/dominia_noble)
+	whitelisted = list(SPECIES_HUMAN, SPECIES_HUMAN_OFFWORLD)
+	cost = 1
+
+/datum/gear/augment/geneboosted_liver
+	display_name = "Geneboosted Liver"
+	description = "A liver that has been enhanced by Imperial geneboosting to function at the level of a peak human athlete."
+	path = /obj/item/organ/internal/liver/geneboosted_liver
+	origin_restriction = list(/singleton/origin_item/origin/dominia_noble)
+	whitelisted = list(SPECIES_HUMAN, SPECIES_HUMAN_OFFWORLD)
+	cost = 1
```

Comment:
Arrow and I have spoken about this. We're fine with these being fluff items (or items with very minor benefits) but we do not want this to be a mechanical buff.

I think the impact of the geneboosted heart is also quite strong as it is honestly.

Reply:
I was considering taking them out of the PR anyways since I actually agreed that they didn't need to be in it anyways. So they're out of it now.